samples/AgentServer/README.md

```diff
@@ -88,4 +94,51 @@ The `http-tests` directory contains HTTP test files that can be executed directl
 - `message-stream.http`: Test message streaming
 - `push-notifications.http`: Test push notifications
 - `researcher-agent.http`: Test researcher agent specific features
-- `task-management.http`: Test task management features
\ No newline at end of file
+- `task-management.http`: Test task management features
+
+## A2A Specification Compliance Verification
+
+The `speccompliance` agent is specifically designed to validate compliance with the A2A specification. You can verify this compliance using the official A2A TCK.
+
+### Prerequisites
+
+1. **Start the speccompliance agent**: Make sure the AgentServer is running with the speccompliance agent (see [Running the Application](#running-the-application) above)
+2. **Install TCK prerequisites**: Follow the [A2A TCK requirements](https://github.com/a2aproject/a2a-tck#requirements) for your platform
+
+### Running the TCK
+
+1. **Clone the A2A TCK repository**:
+   ```bash
+   git clone https://github.com/a2aproject/a2a-tck.git
+   cd a2a-tck
+   ```
+
+2. **Setup Python environment**:
+   ```bash
+   # Create virtual environment
+   uv venv
```

Comment:
Thank you for providing the exact steps, I've spent half an hour this week trying to fight pip to get me the right packages to be able to run tck 👍